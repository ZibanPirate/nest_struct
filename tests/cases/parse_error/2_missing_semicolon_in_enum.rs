#![allow(dead_code)]
#[macro_use]
extern crate nest_struct;

#[nest_struct]
struct NestedEnum {
    id: u32,
    nested_enum: nest! {
        First
        Last,
    },
}

fn main() {}
