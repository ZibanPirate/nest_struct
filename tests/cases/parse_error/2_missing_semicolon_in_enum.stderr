error: if nesting a struct: expected `:`
  --> tests/cases/parse_error/2_missing_semicolon_in_enum.rs:10:9
   |
10 |         Last,
   |         ^^^^

error: if nesting an enum: expected `,`
  --> tests/cases/parse_error/2_missing_semicolon_in_enum.rs:10:9
   |
10 |         Last,
   |         ^^^^

error: if nesting a block: unexpected token, expected `;`
  --> tests/cases/parse_error/2_missing_semicolon_in_enum.rs:10:9
   |
10 |         Last,
   |         ^^^^
